FGEGraphics/UISystem/UIElementStyle.cs

```diff
@@ -0,0 +1,78 @@
+﻿//
+// This file is part of the Frenetic Game Engine, created by Frenetic LLC.
+// This code is Copyright (C) Frenetic LLC under the terms of a strict license.
+// See README.md or LICENSE.txt in the FreneticGameEngine source root for the contents of the license.
+// If neither of these are available, assume that neither you nor anyone other than the copyright holder
+// hold any right or permission to use this software until such time as the official license is identified.
+//
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using FGECore.ConsoleHelpers;
+using FGECore.MathHelpers;
+using FGEGraphics.GraphicsHelpers.FontSets;
+using FGEGraphics.GraphicsHelpers.Textures;
+
+namespace FGEGraphics.UISystem;
+
+/// <summary>Represents the rendering style of a <see cref="UIElement"/>.</summary>
+public record UIElementStyle
+{
+    /// <summary>An empty element style.</summary>
+    public static readonly UIElementStyle Empty = new();
+
+    /// <summary>What base color to use (or <see cref="Color4F.Transparent"/> for none).</summary>
+    public Color4F BaseColor = Color4F.Transparent;
+
+    /// <summary>What texture to display (or <c>null</c> for none).</summary>
+    public Texture BaseTexture;
```

Comment:
I don't know that a hashed record pattern gets along well with remote GPU-allocated dynamic textures. The texture/shader/etc. GPU-allocated asset bind pattern is `BaseTexture = BaseTexture.Bind();` specifically because the object will at random without warning or reason need to replace itself.

Reply:
This isn't used anywhere at the moment, and I think `UITexture` replaces this functionality anyways -- will remove